config/nginx.conf.erb

```diff
@@ -34,6 +34,7 @@ http {
     listen <%= ENV["PORT"] %>;
     server_name _;
     keepalive_timeout 5;
+    port_in_redirect off;
```

Comment:
not sure why this was needed, but hit some weird issue in my side project where django redirects ended up with the internal port stamped in

since heroku load balancer is always gonna be operating on a fixed port, we can always drop the port for redirect